proto/axelar/axelarnet/v1beta1/query.proto

```diff
@@ -21,3 +21,39 @@ message PendingIBCTransferCountResponse {
 message ParamsRequest {}
 
 message ParamsResponse { Params params = 1 [ (gogoproto.nullable) = false ]; }
+
+enum ChainStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  CHAIN_STATUS_UNSPECIFIED = 0
+      [ (gogoproto.enumvalue_customname) = "StatusUnspecified" ];
+  CHAIN_STATUS_ACTIVATED = 1 [ (gogoproto.enumvalue_customname) = "Activated" ];
+  CHAIN_STATUS_DEACTIVATED = 2
+      [ (gogoproto.enumvalue_customname) = "Deactivated" ];
+}
+
+// ChainsRequest represents a message that queries all registered cosmos chains
+// by status
+message ChainsRequest { ChainStatus status = 1; }
+
+message ChainsResponse {
+  repeated string chains = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+}
+
+// IBCPathRequest represents a message that queries the IBC path registered for
+// a given chain
+message IBCPathRequest { string chain = 1; }
```

Comment:
Why's the `chain` here not a `github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName`?

Reply:
the REST api generation via grpc-web can't handle gogoproto custom type/names for the request type. It needs to use primitive types. Mentioned [here](https://github.com/axelarnetwork/axelar-core/tree/main/proto/axelar#rest-api)

Reply:
add that as a comment?

Reply:
it affects many request types which is why i added to the readme previously so it's not verbose